src/app/shell/personal-cabinet/shared-cabinet/applications/applications.component.ts

```diff
@@ -80,30 +67,22 @@ export class ApplicationsComponent implements OnInit, OnDestroy, AfterViewInit {
     this.isMobileView = event.outerWidth < 530;
   }
 
-  constructor(
-    protected store: Store,
-    protected router: Router,
-    protected route: ActivatedRoute,
-    protected actions$: Actions
-  ) {
+  constructor(protected store: Store, protected router: Router, protected route: ActivatedRoute, protected actions$: Actions) {
     this.onResize(window);
   }
 
-  onGetApplications(): void {
-    this.getApplications.emit();
-  }
-
   onEntitiesSelect(IDs: string[]): void {
     this.enititiesSelect.emit(IDs);
   }
 
   ngOnInit(): void {
-    this.searchFormControl.valueChanges
-      .pipe(debounceTime(500), takeUntil(this.destroy$))
-      .subscribe((searchString: string) => {
-        this.applicationParams.searchString = searchString;
-        this.onGetApplications();
-      });
+    Util.setFromPaginationParam(this.applicationParams, this.currentPage);
```

Comment:
I think this is redundant, because this is included in onGetApplications method as well